src/public/publicAPIs.ts

```diff
@@ -393,6 +393,16 @@ export function executeDeepLink(deepLink: string, onComplete?: (status: boolean,
   GlobalVars.callbacks[messageId] = onComplete ? onComplete : getGenericOnCompleteHandler();
 }
 
+/**
+ * Return focus to the main Teams app. Will focus search bar if navigating foward and app bar if navigating back.
+ * @param navigateForward Determines the direction to focus in teams app.
+ */
+export function returnFocus(navigateForward?: boolean): void {
```

Comment:
Should we put under navigation namespace.

Reply:
Added navigation namespace